Tests/_CryptoExtrasTests/TestRSAEncryption.swift

```diff
@@ -36,8 +36,13 @@ final class TestRSAEncryption: XCTestCase {
 
         let derPubKey = derPrivKey.publicKey
 
-        guard group.sha == "SHA-1", group.mgfSha == "SHA-1" else {
-            // We currently only support SHA-1 OAEP, which is very legacy but oh well.
+        let padding: _RSA.Encryption.Padding
```

Comment:
We should add a new set of test vectors: https://github.com/google/wycheproof/blob/master/testvectors/rsa_oaep_2048_sha256_mgf1sha256_test.json